<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QWidget" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainInterface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>8</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="devicesComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="frameDisplayer" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>9</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>600</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>初始化</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="findDevpbt">
              <property name="text">
               <string>查找设备</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="openDevpbt">
                <property name="text">
                 <string>打开设备</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="closeDevpbt">
                <property name="text">
                 <string>关闭设备</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>图像采集</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QRadioButton" name="triggerMode">
              <property name="text">
               <string>触发模式</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="stoppbt">
              <property name="text">
               <string>停止采集</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="tigger1">
              <property name="text">
               <string>软触发一次</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="continuousMode">
              <property name="text">
               <string>连续模式</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="startpbt">
              <property name="text">
               <string>开始采集</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="softTrigger">
              <property name="text">
               <string>软触发</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>图片保存</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPushButton" name="saveBMPpbt">
              <property name="text">
               <string>保存BMP</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="saveJPGpbt">
              <property name="text">
               <string>保存JPG</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="saveTiffpbt">
              <property name="text">
               <string>保存tiff</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="savePNGpbt">
              <property name="text">
               <string>保存PNG</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="styleSheet">
            <string notr="true">QSpinBox
{
	border:1px solid #242424;
}
 
QSpinBox::up-button,QSpinBox::down-button
{
	width:0px;
}</string>
           </property>
           <property name="title">
            <string>参数</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="2" rowspan="2">
             <widget class="QDoubleSpinBox" name="spinGain"/>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="spinFrameRate"/>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="spinPixelFormat"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>像素格式</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="spinExposure"/>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>帧率</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>增益</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="2" colspan="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>曝光</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="getParameter">
              <property name="text">
               <string>参数获取</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QPushButton" name="setParameter">
              <property name="text">
               <string>参数设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>辅助线绘制</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="circle">
              <property name="text">
               <string>圆形辅助线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="line">
              <property name="text">
               <string>线形辅助线</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
